test/context.test.ts

```
@@ -101,7 +102,7 @@ describe("getRouteParams", () => {
 
         for (const { description, path, schema, expected } of testCases) {
             test.concurrent(description, ({ expect }) => {
-                const params = search("GET", root, path).params
+                const params = router.match("GET", path)?.params!
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)